keadm/cmd/keadm/app/cmd/util/common.go

```diff
@@ -425,18 +449,18 @@ func runEdgeCore(version string) error {
 	}
 
 	// add +x for edgecore
-	command := fmt.Sprintf("chmod +x %s/%s", KubeEdgeUsrBinPath, KubeEdgeBinaryName)
+	command := fmt.Sprintf("chmod +x %s/%s", KubeEdgePath, KubeEdgeBinaryName)
 	if _, err := runCommandWithStdout(command); err != nil {
 		return err
 	}
 
 	var binExec string
-	if version >= "1.1.0" {
-		binExec = fmt.Sprintf("%s > %s/%s.log 2>&1 &", KubeEdgeBinaryName, KubeEdgeLogPath, KubeEdgeBinaryName)
-	} else {
-		binExec = fmt.Sprintf("%s > %skubeedge/edge/%s.log 2>&1 &", KubeEdgeBinaryName, KubeEdgePath, KubeEdgeBinaryName)
-	}
-
+	// if version >= "1.1.0" {
+	// 	binExec = fmt.Sprintf("%s > %s/%s.log 2>&1 &", KubeEdgeBinaryName, KubeEdgeLogPath, KubeEdgeBinaryName)
+	// } else {
+	// 	binExec = fmt.Sprintf("%s > %skubeedge/edge/%s.log 2>&1 &", KubeEdgeBinaryName, KubeEdgePath, KubeEdgeBinaryName)
+	// }
```

Comment:
I guess this would break installation support of pre 1.2 releases?
/cc @fisherxu